regression/goto-analyzer/pointer-difference-after-decrement/test.desc

```diff
@@ -0,0 +1,9 @@
+CORE
+main.c
+--variable-sensitivity --vsd-pointers constants --vsd-values constants --verify
+^EXIT=0$
+^SIGNAL=0$
+^\[main.assertion.1\] line 10 assertion q - p == 1: SUCCESS
+^\[main.assertion.2\] line 11 assertion q != p: UNKNOWN
```

Comment:
Why is this unknown?  If the previous one is resolvable it feels like it should be.

Reply:
Turns out the `q-p == 1` case evaluating to success was an artefact. I'm implementing proper pointer comparisons and `q != p` will evaluate properly.